src/main/java/thePackmaster/relics/BanishingDecree.java

```diff
@@ -118,9 +119,14 @@ public void update() {
                 }
 
                 for (int i = 0; i < 3; i++) {
-                    RewardItem r = new RewardItem();
-                    r.cards = getCardsFromPacks(cp.packID, 3);
-                    AbstractDungeon.getCurrRoom().rewards.add(r);
+                    RewardItem reward = new RewardItem();
+                    reward.cards = getCardsFromPacks(cp.packID, reward.cards.size(), AbstractDungeon.cardRng);
```

Comment:
What the `reward.cards.size()` on this line (and the similar one in PMBoosterBoxCardReward) is doing:

The base game has two ways that the number of cards in a reward can be modified (relics like Question Card/Busted Crown and the custom run modifier Binary). You could simply reproduce that logic to determine the number of cards to have in a reward, and that would work fine.

However, what if a mod adds some new thing that modifiers card rewards? They'll have patched the logic in `AbstractDungeon.getRewardCards`, but if you've made your own method that has the same logic, there's no way some other mod will know about and patch that method.

So since we're a bit paranoid, we generate a RewardItem and check how many cards there are, and then just use that number for our custom card reward.

On a separate note, the card reward for Banishing Decree does not need to be a registered custom card reward, because there's no need for it to save/load. If you reload after getting Banishing Decree but before going to the next room, you'll need to pick up Banishing Decree again and that will create the card reward again.